diff --git a/setup.py b/setup.py
index f4e328a..7f94557 100755
--- a/setup.py
+++ b/setup.py
@@ -193,7 +193,7 @@ def pkgconfig(*packages_options, **ekw):
     WARN_ALL = True
     if WARN_ALL:
         add_to_keywords(kw, 'extra_compile_args', "-Wall")
-        add_to_keywords(kw, 'extra_link_args', "-Wall")
+	add_to_keywords(kw, 'extra_link_args', "-Wall", "-lX11", "-lm", "-lavformat", "-lavcodec", "-lavutil")
     PIC = True
     if PIC:
         add_to_keywords(kw, 'extra_compile_args', "-fPIC")
