--- xpra-0.16.2/setup.py.omv~	2016-02-29 03:22:52.239874758 +0100
+++ xpra-0.16.2/setup.py	2016-02-29 03:24:14.284378361 +0100
@@ -603,7 +603,7 @@ def exec_pkgconfig(*pkgs_options, **ekw)
             add_to_keywords(kw, 'extra_compile_args', "/WX")
             add_to_keywords(kw, 'extra_link_args', "/WX")
         else:
-            if os.environ.get("CC", "").find("clang")>=0:
+            if os.environ.get("CC", "").find("gcc")<0:
                 #clang emits too many warnings with cython code,
                 #so we can't enable Werror
                 eifd = ["-Werror",
@@ -1880,7 +1880,7 @@ if client_ENABLED and PYTHON3:
     #cairo workaround:
     cython_add(Extension("xpra.client.gtk3.cairo_workaround",
                 ["xpra/client/gtk3/cairo_workaround.pyx", buffers_c],
-                **pkgconfig("pycairo")
+                **pkgconfig("py3cairo")
                 ))
 
 add_packages("xpra.codecs.argb")
